tmt/utils.py

```diff
@@ -681,6 +698,7 @@ def log_event(msg: str) -> None:
 
     def run(self,
             command: Union[str, List[str]],
+            printable_command: Optional[str] = None,
```

Comment:
Isn't the new `printable_command` parameter here intended for the same/similar purpose as `message` below? We could just move it from `debug` to `verbose`. So far, it seems, `message` hasn't been used much.

Reply:
Do you mean present `message` with `cmd` prefix in the output? I am find with that, however I don't know what are the other use cases. I like the `cmd` prefix in the output which makes it understandable.

Reply:
I have also decided to preserve `message` since it makes sense to have such possibility. However, I would prefer moving it from `debug` to `verbose` mode.

Reply:
Ack, makes sense to keep `message` for human readable sentences describing what's happenning.